src/pair.py

```diff
@@ -3,3 +3,66 @@
 # Part of azplugins, released under the BSD 3-Clause License.
 
 """Pair potentials."""
+
+from hoomd.azplugins import _azplugins
+from hoomd.data.parameterdicts import TypeParameterDict
+from hoomd.data.typeparam import TypeParameter
+from hoomd.md import pair
+
+
+class Hertz(pair.Pair):
+    r"""Hertz potential.
+
+    Args:
+        nlist (hoomd.md.nlist.NeighborList): Neighbor list.
+        r_cut (float): Default cutoff radius :math:`[\mathrm{length}]`.
+        mode (str): Energy shifting/smoothing mode.
+
+    :py:class:`hertz` is the Hertz potential:
+
+    .. math::
+        :nowrap:
+
+        \begin{eqnarray*}
+        V(r)  &= \varepsilon ( 1-\frac{ r }{ r_{\rm{cut}} } )^{5/2} ,
+                & r < r_{\rm{cut}} \\
+              &= 0,& r \ge r_{\rm{cut}}
+        \end{eqnarray*}
+
+    Example::
+
+        nl = hoomd.md.nlist.cell()
+        hertz = azplugins.pair.hertz(r_cut=3.0, nlist=nl)
```

Comment:
The hoomd4 convention appears to be to capitalize potentials now, so if we want to be consistent with hoomd4 naming scheme, it might make sense to cal this potential `Hertzian` instead of `hertz`. See: https://hoomd-blue.readthedocs.io/en/v4.7.0/module-md-pair.html - This change would only affects the python naming scheme.

Reply:
The lowercase "hertz" was a typo. I've corrected those to "Hertz" to be consistent with hoomd4's naming scheme. I think we should be okay keeping Hertz rather than Hertzian.

Reply:
Yep, sounds good!